### DISCUSSION TITLE
Compatibility with tweakeroos freecam

### DISCUSSION THREAD

**Discussion by Joulanh** [SchmarrnDevs/Lighty#83] [Ideas and Suggestions]
It would be really nice to render the overlay "around" the tweakeroo freecam when in use to check for dark spots in hard to reach places.

Another feature could be the option to switch from the cylindrical rendering to a spherical one for some fps gains, but I'm not sure if the benefit is even big enough to make the effort of implementing it.

**Comment by andi-makes** [comment]
Thanks for your suggestions!

I'll consider the camera suggestion, but I'll have to look into how well the current code would adapt to such a change.

Regarding cylindrical vs spherical rendering, there won't be much FPS gains. Do you experience big performance issues, or would that just be a nice to have?

**Reply by Joulanh** [reply]
Nah the spherical rendering was just an afterthought the main thing is the freecam. 

Thanks for the reply :)

**Comment by andi-makes** [comment]
Hi! Lighty v3.0.0-beta.8 now renders the overlay around the players camera, so the overlay follows you around now when you are in freecam! (didn't test it with tweakeroos freecam, but with freecam and it worked there)

Thanks again for your suggestion and have fun!
